backend/src/user/user_utils.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * user_utils.ts
```

Comment:
as mentioned earlier,
```suggestion
 * user-utils.ts
```